### ISSUE TITLE
feat: cache representations

### ISSUE THREAD

**Issue by devversion** [DevVersion/vpapp-ionic2#6]
We should create a service which stores all representations for 2-3 days (maybe you find a better delay).

Those should be stored in ionics - SQL Storage and also backend up into the local storage for better performance.

Take a look at the SessionAccessor.

@Lordnoname

**Comment by thielpa** [comment]
@DevVersion  Doesn't it make more sense, if we store only the recent representations from the user sesssion (today & tomorrow)? If the user has no internet connection, he is going to have the same representations as before. Why do you want to store older representations or all representations (uses more data volume; each user has only one account)?

**Comment by devversion** [comment]
@Lordnoname Agree, I just thought about 2-3 days because of the weekends (there we will have a delay of max-3 days).

But if you could manage to always cache the current representations for the `today` and `tomorrow` date from the dateUtils, would be great.

But as mentioned, it should be stored in the SQL Storage and Local Storage, as same as in the SessionAccessor.

**Comment by thielpa** [comment]
Sure, I'll finish that today. Then are only the svgs missing and we're finished, aren't we?

**Comment by pgschwendtner** [comment]
Thanks for your reply, we will analyze your comment as soon as possible.
